module-2/app.ts

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import * as dotenv from 'dotenv';
+import userRouter from './routes/userRouter';
+import { errorHandler, pageNotFound } from './middleware/errorHandlers';
+
+dotenv.config();
```

Comment:
In the future if you get errors accessing env file, then move this line to the top

Reply:
like this

import * as dotenv from 'dotenv';
dotenv.config();
import express from 'express';